.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,11 +2,18 @@
 v                               ✰  Thanks for creating a PR! ✰
 ☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
 
-<!-- Most PRs should close a specific Issue. All PRs should at least reference one or more Issues. Edit and/or delete the following lines as appropriate (note: you don't need both `refs` and `closes` for the same one): -->
+<!-- Most PRs should close a specific issue. All PRs should at least reference one or more issues. Edit and/or delete the following lines as appropriate (note: you don't need both `refs` and `closes` for the same one): -->
 
 closes: #XXXX
 refs: #XXXX
 
+<!-- Integration testing doesn't run for every PR, but can be opted into by adding label 'force:integration', and can be customized to use non-default external references by including lines here that **start** with leading-`#` directives:
```

Comment:
This is helpful. How can we ensure that it's updated when the truth of this changes?

Maybe if each branch override config has a reference to this file? Alternately we could adopt a scheme like `#integrate: <repo> <branch>`

Not asking for that work just exploring options

Reply:
I don't think we need process here; changes to the list of directives are rare and the template is implicitly reviewed roughly once per PR, so I have confidence that it would catch up in short order. But if we do DRY out something like .github/actions/read-pr-directive, I could see adding a link from its description to this comment in the template.

Reply:
Integration testing does run for every PR, but only once the PR is labelled to merge. `force:integration` allows running integration tests for every PR push.

Reply:
Arguably the external references should be removed before merging the PR to verify that the integration with the default branch still works (our logic is not sufficiently sophisticated to ignore these directives at the time of merge, plus it's possible we might someday do synchronized merges across repos which requires one side to keep the directive through merging)